spec/services/people/retrieve_image_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe People::RetrieveImage do
+  it 'returns false if latest_nomis_booking_id is empty' do
+    person = build(:person)
```

Comment:
how about a `let(:person) { build(:person) } ` outside of the test, and the test can just check if something is true or not

Reply:
The test I wrote has a different "style", it follows the simple AAA Assign, Act, Assert that maximize readability and avoid all the rspec DSL, 
but I agree with you that since we are using a certain style, mainly based on http://www.betterspecs.org/ (DRY and fragmented style), 
I should adapt the code.

Reply:
👍 yep, you are right, it is purely a style issue - it would be good if to be consistent with other tests

Reply:
I tried to mix up the best of the 2 approaches, I dried it without making it too fragmented